test/tests.js

```diff
@@ -1107,6 +1117,45 @@ describe('Braze Forwarder', function() {
         (window.braze.getUser().yearOfBirth === null).should.equal(true);
     });
 
+    it('it should set subscription group for mapped attributes when value is boolean', function() {
```

Comment:
no need for the line to start with `it`

```suggestion
    it('should set subscription group for mapped attributes when value is boolean', function() {
```